client/components/PubEdgeListing/PubEdgeListingCard.js

```diff
@@ -0,0 +1,57 @@
+import { Icon } from '@blueprintjs/core';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { PubEdge } from 'components';
+import { toTitleCase } from '../../utils/string';
```

Comment:
Prefer `client/utils/string` here — even though it breaks my editor's ability to infer anything from the file. Also, since this is isomorphic JS, I'd encourage you to move it to the top-level `utils` folder, where I am sure these will see repeated use!